site/_data/versions.yml

```diff
@@ -0,0 +1,4 @@
+vega: 5.22.1
+vega-lite: 5.5.0
+vega-embed: 6.21.0
+vega-tooltip: 0.28.0
```

Comment:
I'm not sure if I should include this file.

Reply:
yeah, please remove it. It doesn't seem relevant to this PR.